test/integration/quic_protocol_integration_test.cc

```diff
@@ -2,13 +2,13 @@
 
 namespace Envoy {
 
-// These will run with HTTP/3 downstream, and Http and HTTP/2 upstream.
-INSTANTIATE_TEST_SUITE_P(Protocols, DownstreamProtocolIntegrationTest,
+// These will run with HTTP/3 downstream, and Http upstream.
```

Comment:
why remove HTTP/2?

Reply:
instantiate_protocol_integration_test.cc also only run DownstreamProtocolIntegrationTest with HTTP1 upstream. And RetryPriority do fail with H2 upstream but not H1 with any downstream protocol.